scripts/conan/create_vs_solution.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+
+import argparse
+from pathlib import Path
+import platform
+import subprocess
+import sys
+
+from conans.client import conan_api
+
+
+openrw_path = Path(__file__).resolve().parents[2]
+windows_profile = openrw_path / 'scripts' / 'conan' / 'windows'
+cmake_generator_lookup = {
+    2015: 'Visual Studio 14 2015',
+    2017: 'Visual Studio 15 2017',
```

Comment:
Doesn't VS 2017 have native CMake support?

Reply:
Yes, it has. See [this link](https://blogs.msdn.microsoft.com/vcblog/2016/10/05/cmake-support-in-visual-studio/)
I don't see immediately a way to run conan there.